openlibrary/plugins/openlibrary/api.py

```diff
@@ -66,6 +66,11 @@ def GET(self):
         sorts = i.sorts.split(',')
         page = int(i.page)
         limit = int(i.limit)
+
+        print("query: " + i.q)
+        print("subject: " + i.subject)
+        print("work_id: " + i.work_id)
```

Comment:
Should these remain?  Should they be `logger.info()` instead?

Reply:
Nope!